src/Core/MailTemplate/ThemeCollection.php

```diff
@@ -40,4 +40,21 @@ protected function getType()
     {
         return ThemeInterface::class;
     }
+
+    /**
+     * @param string $themeName
+     *
+     * @return ThemeInterface|null
+     */
+    public function getByName($themeName)
+    {
+        /** @var ThemeInterface $theme */
+        foreach ($this as $theme) {
```

Comment:
Try to no use magic as must as possible.
```suggestion
        foreach ($this->getValues() as $theme) {
```

Reply:
Done